File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.py
def solution(my_string, s, e):
    my_lst = list(my_string[s:e+1])
    re_lst = my_lst[::-1]
    
    answer = my_string[:s] +  ''.join(re_lst) + my_string[e+1:]
    
    return answer

File: 프로그래머스/0/181905. 문자열 뒤집기/문자열 뒤집기.js
function solution(my_string, s, e) {
    var answer = '';
    var left = my_string.slice(0, s);
    var middle = my_string.slice(s, e + 1);
    var right = my_string.slice(e+1);
    answer = left + [...middle].reverse().join('') + right;
    
    return answer;
}